### PR TITLE
fix: resolve issue with logo click not redirecting to home

### LINKED ISSUE #74: [BUG] App logo should be link that redirect to home page
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

both

### Description

Header logo of the website should be a link that redirect to home page.

### Steps to Reproduce

If you know the steps, follow the below format and provide steps to reproduce

For example:

Go to https://repo-rater.eddiehub.io/popular page.
And click on icon on the header

### Screenshots

image

### Do you want to work on this issue?

No

### If "yes" to above, please explain how you would technically implement this

_No response_

### PR DESCRIPTION
**Pull request by Nithishvb**
closes #74

This PR addresses a bug where clicking the app logo did not correctly redirect to the home page. The issue has been identified and fixed. Now, when users click the logo, it successfully redirects to the home page.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ export default function Navbar() {
       <div className="navbar bg-base-100">
         <div className="flex-1">
           <div className="flex flex-row">
-            <Image src={Logo} alt="RepoRater Logo" width={40} height={40} />
+            <Image src={Logo} alt="RepoRater Logo" width={40} height={40} className="cursor-pointer" onClick={() => router.push("/")} />
```
Comment (eddiejaoude) [inline src/components/Navbar.js:49]: What about wrapping it in a `Link` component?

Comment (Nithishvb) [reply src/components/Navbar.js:49]: sure, i will update the pull request with the requested changes.

**Comment by Nithishvb** [comment]
Hi @eddiejaoude updated this PR with the link tag and removed the previous changes.

**Review by j-grosse — APPROVED** [review]
The image is now wrapped inside of a Link tag. The code looks fine and can now be merged.

**Comment by Nithishvb** [comment]
Hi @j-grosse  thanks for the review , I have now deleted the unnecessary "useRouter" and updated this PR.

**Review by j-grosse — APPROVED** [review]

**Review by j-grosse — APPROVED** [review]
PR is now mergeable

File: src/components/Navbar.js (hunk 1 above)
Comment (eddiejaoude) [reply src/components/Navbar.js]: Thanks 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍  Great collaboration 💪

**Pull request merged**